tests/e2e/playwright.config.ts

```diff
@@ -15,15 +15,26 @@ const vscodeVersion = process.env.VSCODE_VERSION || "stable";
 export default defineConfig({
   testDir: path.join(__dirname, "specs"),
   forbidOnly: !!process.env.CI,
-  retries: process.env.CI ? 2 : 0,
+  retries: 0,
```

Comment:
No need for this since the singular test will either pass or not, and doesn't depend on CCloud auth callbacks or any longer test(s)